popde/density_estimate.py

```diff
@@ -362,6 +360,30 @@ class MultiDimRescalingBwKDEPy(VariableBwKDEPy):
     --------
     TBA
     """
-    % WIP
-    pass
-    
+    def __init__(self, data, input_transf=None, stdize=False, rescale=None,
+                 backend='KDEpy', bandwidth=1., dim_names=None,
+                 bandwidth_method='oned_isj')
+        """
+        bandwidth method: string
+           Name of method to get bandwidths for each dimension of the KDE
+
+        A general diagonal covariance Gaussian kernel can be written 
+          K(x - X) = const. * (h_1 h_2 ...)^(-1) exp(-(x - X)^T . diag(h_1^2, h_2^2, ...)^(-1) . (x - X) / 2)
+        This may be implemented by changing variables to w_i = x_i / h_i and 
+        using a unit kernel matrix :
+          K(w - W) = const. * exp(-(w - W)^T . diag(1, 1, ...) . (w - W) / 2).
+        More generally, the kernel in 'w' units should be proportional to the
+        unit matrix (e.g. it may vary between data points).
+        """
+        # Check compatibility of input options
+        if stdize:
```

Comment:
why standardization i not allowed?

Reply:
Because it would not do anything more than the rescaling we are implementing with the bw method
ie we would standardize then compute a bw for each dimension and rescale, but then we would get the same outcome as if we computed the bw on the data without standardizing 
(since they are both just multiplying by a constant factor)